code/Items/Data/Item.cs

```diff
@@ -5,11 +5,21 @@ namespace ssl.Items.Data
 {
     public abstract class Item
     {
-        public abstract string Id { get; } 
-        public abstract string Name { get; } 
-        public abstract int MaxStack { get; } 
-        public abstract bool DestroyOnUse { get; }
-        public abstract string Model { get; }
+        protected Item(string id, string name, string model, int maxStack, bool destroyOnUse)
+        {
+            Id = id;
+            Name = name;
+            Model = model;
+            MaxStack = maxStack;
+            DestroyOnUse = destroyOnUse;
+        }
+
+        public string Id { get; protected set; } 
+        public string Name { get; protected set; }
+        public string Model { get; protected set; }
```

Comment:
On sais pas ce qu'est le modele on peux le delete for now

Reply:
Bah on aura forcément besoin de donner le model 3d à un moment, c'était le but de ma mr d'implémentation des objets dans S&box

Reply:
C'est vrai. Je te le concède :)